src/chromium/interoperability-with-cpp/example-bindings.md

```diff
@@ -1,6 +1,6 @@
 # Example bindings
 
-cxx requires you to declare the whole C++/Rust boundary in one of your `.rs`
+cxx requires you to declare the C++/Rust boundary in one of your `.rs`
```

Comment:
I think that saying "whole" is not 100% correct.  It's possible (and sometimes okay / desirable) to have more than one `cxx::bridge` in a crate (e.g. for 2 non-overlapping areas of interop).

I think it's safest to just drop "whole", to avoid the potential to mislead.

Reply:
I suppose you're technically correct, but I think the spirit of the word "whole" still stands - the entire Rust/C++ interop surface needs to be in a `cxx::bridge` somewhere. Perhaps it could be worded like that:

> cxx requires that the whole C++/Rust boundary is declared in `cxx::bridge` mods within a `.rs`

Reply:
Done, with some minimal rewording